src/routes/busRoutes.js

```diff
@@ -0,0 +1,44 @@
+import busController from '../controller/busControllers'
+import busValidation from '../validation/busValidation'
+import checkExist from '../middleware/checkExist'
+import protectMiddleware from '../middleware/protectRoutes'
+import express from "express"
+
+const router = express()
+
+router
+    .route('/')
+    .get(
+        protectMiddleware.protect,
+        protectMiddleware.restrictTo('operator', 'admin'),
+        busController.getAllBus
+    )
+    .post(
+        protectMiddleware.protect,
+        protectMiddleware.restrictTo('operator', 'admin'),
+        busValidation.busValidate,
+        checkExist.checkPlate,
+        busController.createNewBus
+    )
+
+router
+    .route('/:id')
+    .get(
+        checkExist.checkID,
+        busController.getBus
```

Comment:
put your swagger documentation always on the top of a route
not in the swagger.json file

Reply:
Done